include/mbedtls/ssl.h

```diff
@@ -646,9 +653,21 @@ typedef int mbedtls_ssl_async_sign_t( mbedtls_ssl_context *ssl,
  *                  store an operation context for later retrieval
  *                  by the resume callback.
  *
+ * \warning         RSA decryption as used in TLS is subject to a potential
+ *                  timing side channel attack first discovered by Bleichenbacher
+ *                  in 1998. This attack can be remotely exploitable
```

Comment:
Nitpicking: IIRC Bleichenbacher's original attack was not about timing, but about different error messages (I think SSLv3 even allowed/encouraged sending different alert messages). It was only later extended to rely on timing. Not that it matters in practice, so the documentation is fine as it is.